bentoml/_internal/cli/click_utils.py

```diff
@@ -181,18 +194,9 @@ def wrapper(func):
         return wrapper
 
 
-def _is_valid_bento_tag(value):
+def is_valid_bento_tag(value: str) -> bool:
```

Comment:
should we just use the Tag class's validation logic here?

Reply:
yep we should